### ISSUE TITLE
Option to disable colors in titles

### ISSUE THREAD

**Issue by kuba-orlik** [schaal/ocreader#12]
In current state, the color of the text of the article's title reflects the color of the feed's favicon. Some favicons, unfortunately result in colors that are too bright and are therefore close unreadable on a white background. I suggest two ways to fix this issue: 

* limit the lightness of the color - one could convert it to hsl, and clip the lightness component at 50%
* add option to disable colorful article titles

**Comment by Mattia98** [comment]
You could also use black text when it gets to bright.

**Comment by schaal** [comment]
@kuba-orlik Could you share the link to the affected feeds?

**Comment by kuba-orlik** [comment]
http://hackernews.io-meter.com/rss
http://feeds.gawker.com/lifehacker/full

**Comment by schaal** [comment]
Turns out there are two different issues here:

1. http://hackernews.io-meter.com/rss doesn't have a favicon, so the app just chooses a random color from a predefined list, I just changed this list to use darker colors
2. http://feeds.gawker.com/lifehacker/full has a favicon, but the contrast of its color is too low, I changed the color extraction code to discard colors with too little contrast against the background.
